### PR TITLE
Test for #1320 - Description produced by Request.classes() shouldn't be null

### PR DESCRIPTION
**Pull request by PeterWippermann**

### CODE REVIEW

File: src/test/java/org/junit/runner/RequestTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package org.junit.runner;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+public class RequestTest {
```
Comment (kcooney) [inline src/test/java/org/junit/runner/RequestTest.java]: You need to add RequestTest.class to src/test/java/org/junit/runner/AllRunnerTests.java in order for the Maven build to run your test.

Please verify the test fails when the changes you made to Computer are removed

Comment (PeterWippermann) [inline src/test/java/org/junit/runner/RequestTest.java]: done :-)

**Comment by kcooney** [comment]
Thanks! Could you please update the release notes?

**Comment by PeterWippermann** [comment]
Added release notes.

**Pull request merged**